content/blog/2016/2016-05-18-external-workspace-manager-plugin.adoc

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: "GSoC Project Intro. External Workspace Manager Plugin"
+tags:
+- jenkins
+- pipeline
+- plugins
+- gsoc
+author: alexsomai
+---
+
+=== About myself
+My name is Alexandru Somai.
+I am following a major in Software Engineering at the Babes-Bolyai University of Cluj-Napoca, Romania.
+I have more than two years hands-on experience working in Software Development.
+I am always learning, always looking for new challenges.
```

Comment:
NIT: Jenkins is a developer community, so maybe several words about your background (languages, technologies, tools) would be useful